pkg/features/patch_kube_features.go

```diff
@@ -0,0 +1,27 @@
+package features
+
+import (
+	"os"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/klog/v2"
+)
+
+// This override should only be used with a support exception. It provides a way
+// to set the NewVolumeManagerReconstruction feature gate in a way that still
+// allows the cluster to be upgradable. This is only needed on 4.14, since the
+// feature gate defaults to true on 4.15 and later. Owners: @jsafrane @dobsonj
+func OpenShiftNewVolumeManagerReconstructionOverride() {
+	overrideVariable := "OCP_4_14_SUPPORT_EXCEPTION_ENABLE_NEW_VOLUME_MANAGER_RECONSTRUCTION"
+	_, enabled := os.LookupEnv(overrideVariable)
+	if enabled {
+		klog.Infof("Environment variable %s is set, setting feature gate %s to true", overrideVariable, string(NewVolumeManagerReconstruction))
+		fg := map[string]bool{string(NewVolumeManagerReconstruction): true}
+		runtime.Must(utilfeature.DefaultMutableFeatureGate.SetFromMap(fg))
+	}
+}
+
+func init() {
+	OpenShiftNewVolumeManagerReconstructionOverride()
+}
```

Comment:
In case anyone else has the same question I had: What happens when two files in a single package both have their own `init()` routine?

The files are processed alphabetically. First the init() in [kube_features.go](https://github.com/openshift/kubernetes/blob/401bb488d6f5e704de86bbc28fcdd907a130be70/pkg/features/kube_features.go#L921-L923) is called, and then the init() here in `patch_kube_features.go`. They are always called exactly once, in serial, in order.

In other words, `OpenShiftNewVolumeManagerReconstructionOverride()` should always be called after the default feature gates are added.

Reply:
It would be nice to have this information in the code.